src/main/java/org/folio/edge/sip2/repositories/CirculationRepository.java

```diff
@@ -82,6 +82,8 @@
               .transactionDate(OffsetDateTime.now(clock))
               .institutionId(institutionId)
               .itemIdentifier(itemIdentifier)
+              .titleIdentifier(resource.getResource() == null ? null
```

Comment:
We may want to ask Bibliotheca on what to return if the title identifier is null so that the kiosk errors out gracefully?

Reply:
Good call. I asked them via the support ticket and will wait for their response.

Ultimately, a missing title is likely either a bug in FOLIO or someone is messing with the database. The `title` field is required for instances, so it should always be there. In this case, even though the Kiosk says checkin failed, the checkin actually succeeded in FOLIO. Maybe this would prompt a patron to take the book to a service point for further evaluation. The staff could discover the root cause and, perhaps, correct it. So, I don't mind making it less user friendly in this case. 😏

Anyway, we'll see what Bibliotheca recommends.

Reply:
Bibliotheca recommends that we put the item identifier in the `AJ` field as the title for the rare case where the `title` field is missing in the response from FOLIO, so I'll do that as part of this PR.